pkg/build/api/v1/types.go

```diff
@@ -303,6 +303,10 @@ type BuildStrategy struct {
 
 	// CustomStrategy holds the parameters to the Custom build strategy
 	CustomStrategy *CustomBuildStrategy `json:"customStrategy,omitempty"`
+
+	// JenkinsPipelineStrategy holds the parameters to the Jenkins Pipeline build strategy.
+	// This strategy is experimental.
```

Comment:
We have a draft of an image, but we need rpm versions of a bunch of plugins before we can get a rhel version produced.  and i think we're still pretty far from the finalized version of the plugin that's needed in the jenkins image (in terms of how it gets configured/etc by the user)